bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/SitemapSubscriptionService.java

```diff
@@ -215,7 +216,7 @@ private void addCallbackToListener(String sitemapName, String pageId, SitemapSub
     }
 
     private EList<Widget> collectWidgets(String sitemapName, String pageId) {
-        EList<Widget> widgets = null;
+        EList<Widget> widgets = new BasicEList<Widget>();
```

Comment:
The variable in line [205](https://github.com/eclipse/smarthome/blob/master/bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/SitemapSubscriptionService.java#L205) is just a local one and there is a check for `null` before the function does something with this `null` list .